gitea/tests/docker/app.ini

```diff
@@ -0,0 +1,90 @@
+APP_NAME = Gitea: Git with a cup of tea
+RUN_MODE = prod
+RUN_USER = git
+
+[repository]
+ROOT = /data/git/repositories
+
+[repository.local]
+LOCAL_COPY_PATH = /data/gitea/tmp/local-repo
+
+[repository.upload]
+TEMP_PATH = /data/gitea/uploads
+
+[server]
+APP_DATA_PATH    = /data/gitea
+DOMAIN           = localhost
+SSH_DOMAIN       = localhost
+HTTP_PORT        = 3000
+ROOT_URL         = http://localhost:3000/
+DISABLE_SSH      = false
+SSH_PORT         = 22
+SSH_LISTEN_PORT  = 22
+LFS_START_SERVER = true
+LFS_CONTENT_PATH = /data/git/lfs
+LFS_JWT_SECRET   = dD1IQ2gjlS5FXs3VmxSpav46ni9WHbgkW29OEzD0I28
```

Comment:
Is this secret sensitive or is it ok to use in ddev/CI?

Reply:
This value is randomly generated by Gitea when you setup the instance so we are safe to use it in ddev/CLI. 

I needed to mount this file because the metrics are not exposed by default, so I simply added the `metrics` section.